internal/rollbar/rollbar.go

```diff
@@ -112,6 +111,13 @@ func logToRollbar(critical bool, message string, args ...interface{}) {
 	if len(logData) == logging.TailSize {
 		logData = "<truncated>\n" + logData
 	}
+	if svcLogData := logging.ReadSvcTail(); svcLogData != logging.SvcTailProviderNotSet {
```

Comment:
My thought was to have the value included for explicitness in case something is going wrong somewhere, so please consider including the SvcTailProviderNotSet msg in the logData.

Reply:
Fair enough.